students/uss_tomasz/lesson_01_basics/nowy.py

```diff
@@ -0,0 +1,114 @@
+from math import ceil
+'''
+https://snakify.org/lessons/print_input_numbers/problems/
+'''
+# Problem Sum of three numbers
+number_1 = int(input("First number = "))
+number_2 = int(input("Second number = "))
+number_3 = int(input("Third number = "))
+
+print("Sum of them is", number_1 + number_2 + number_3)
+
+# Problem Hi John
+name = input("What is your name?")
+print("Hi", name)
+
+# Problem Square
+variable = int(input("Input variable = "))
+print("Squared = ", variable ** 2)
+
+# Problem Area of right-angled triangle
+baseLenght = float(input("Length of triangle base = "))
```

Comment:
read PEP8 guidelines regarding variable names and adapt the code accordingly
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names